lib/rpmfi.c

```diff
@@ -2426,7 +2426,7 @@ char * rpmfileStrerror(int rc)
     case RPMERR_DIGEST_MISMATCH: s = _("Digest mismatch");	break;
     case RPMERR_INTERNAL:	s = _("Internal error");	break;
     case RPMERR_UNMAPPED_FILE:	s = _("Archive file not in header"); break;
-    case RPMERR_INVALID_SYMLINK: s = _("Invalid symlink");	break;
+    case RPMERR_INVALID_SYMLINK: s = _("Invalid or unsave symlink");	break;
```

Comment:
s/unsave/unsafe/ although we can probably make the error message more informative yet. The sole purpose of RPMERR_INVALID_SYMLINK is to represent the misowned symlink concept that doesn't exist on the OS level where broken symlinks can be sufficiently expressed with errno stuff, so we should try to explain that in the message. The existing unused "invalid symlink" message was just something I tossed in there to avoid getting stuck on it.

Reply:
So ... actually just "Unsafe symlink" should be quite self-explanatory, once you go and look at the actual path we report.